packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -125,6 +129,9 @@ class WebView extends JavaObject {
   /// The [WebSettings] object used to control the settings for this WebView.
   late final WebSettings settings = WebSettings(this);
 
+  /// The [ScrollChangedCallback] object used to listen for scroll changed events.
+  late ScrollChangedCallback? onScrollChanged;
```

Comment:
This shouldn't be a `late` field. It should be set in both constructors and added to the `copy` method.